.github/workflows/release-please.yml

```diff
@@ -8,6 +9,11 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        id: setup-node
+        with:
+          node-version-file: '.nvmrc'
```

Comment:
The `google-github-actions/release-please-action` action uses node, so I think we want to keep these steps after that.